src/wp-includes/class-wp-image-editor-gd.php

```diff
@@ -296,7 +298,11 @@ public function make_subsize( $size_data ) {
 		if ( is_wp_error( $resized ) ) {
 			$saved = $resized;
 		} else {
-			$saved = $this->_save( $resized );
+			if ( $mime_type ) {
+				$saved = $this->_save( $resized, null, $mime_type );
+			} else {
+				$saved = $this->_save( $resized );
+			}
```

Comment:
The default for `$mime_type` on the `_save()` method is also `null`, so we can simplify this to just
```suggestion
			$saved = $this->_save( $resized, null, $mime_type );
```